app/dashboard/src/layouts/CategorySwitcher.tsx

```diff
@@ -284,8 +290,25 @@ export default function CategorySwitcher(props: CategorySwitcherProps) {
                   <div className="ml-[15px] mr-1 border-r border-primary/20" />
                   {element}
                 </div>
-              ) : (
+              ) : data.category !== Category.local ? (
                 element
+              ) : (
+                <div key={data.category} className="group flex items-center self-stretch">
+                  {element}
+                  <div className="grow" />
```

Comment:
why do we need this empty div here? it looks like we can put `ml-auto` on the button instead?

Reply:
> it looks like we can put `ml-auto` on the button instead?

yeah i guess i'm just not a fan of `m*-auto` to get layouts to work - i'll try switching to `justify-between` first, and switch to `ml-auto` if that doesn't work